content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -260,6 +260,38 @@ problems, the kubelet resets the restart backoff timer for that container.
 [Sidecar containers and Pod lifecycle](/docs/concepts/workloads/pods/sidecar-containers/#sidecar-containers-and-pod-lifecycle)
 explains the behaviour of `init containers` when specify `restartpolicy` field on it.
 
+### Configurable container restart delay
+
+{{< feature-state for_k8s_version="v1.32" state="alpha" >}}
+
+With the alpha feature gate `KubeletCrashLoopBackOffMax` enabled, you can
+reconfigure the maximum delay between container restarts from the default of
+300s (5 minutes). This configuration is set per node using kubelet
+configuration. In your [kubelet
+configuration](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/),
+set the `CrashLoopBackOff.MaxContainerRestartPeriod` field between `"1s"` and
```

Comment:
Please double check the spelling of the field names.
My guess is that the field names starts with lower case letters.
It is upper case because we are programming using GoLang.
What we document here is what the user sees and uses.

```suggestion
set the `CrashLoopBackOff.MaxContainerRestartPeriod` field between `"1s"` and
```